src/components/Filters.tsx

```diff
@@ -93,115 +180,78 @@ const Filters: React.FC<FiltersProps> = ({
-      Sources
-    </Typography>
-    <Box sx={{marginBottom: '-7px'}}>
-    <FormControlLabel
-      control={
-        <Checkbox
-          checked={selectedSources.includes('Demo')}
-          onChange={() => handleSourcesChange('Demo')}
-        />
-      }
-      label="Demo"
-      sx={{ marginBottom: '-7px' }}
-    />
-    </Box>
-    <Box sx={{marginBottom: '-7px'}}>
-    <FormControlLabel
-      control={
-        <Checkbox
-          checked={selectedSources.includes('Code')}
-          onChange={() => handleSourcesChange('Code')}
-        />
-      }
-      label="Code"
-      sx={{ marginBottom: '-7px' }}
-    />
-    </Box>
-    <Box sx={{marginBottom: '-7px'}}>
-    <FormControlLabel
-      control={
-        <Checkbox
-          checked={selectedSources.includes('Tutorial')}
-          onChange={() => handleSourcesChange('Tutorial')}
-        />
-      }
-      label="Tutorial"
-      sx={{ marginBottom: '-7px' }}
-    />
-    </Box>
-  </Box>
+      <Grid item>
+        <Box>
+          <Typography variant="h6" gutterBottom sx={{ paddingLeft: '-12px' }}>
+            Sources
+          </Typography>
+          {['Demo', 'Code', 'Tutorial'].map((source) => (
+            <Box key={source} sx={{ marginBottom: '-7px' }}>
+              <FormControlLabel
+                control={
+                  <Checkbox
+                    checked={selectedSources.includes(source)}
+                    onChange={() => handleSourcesChange(source)}
+                    color="primary"
+                  />
+                }
+                label={source}
+              />
+            </Box>
+          ))}
+        </Box>
       </Grid>
 
       {/* Paratimes */}
-      <Grid item xs={12} md={2}>
-  <Box>
-    <Typography variant="h6" gutterBottom sx={{ paddingLeft: '-12px' }}>
-      Paratimes
-    </Typography>
-    <Box sx={{marginBottom: '-7px'}}>
-    <FormControlLabel
-      control={
-        <Checkbox
-          checked={selectedParatimes.includes('sapphire')}
-          onChange={() => handleParatimesChange('sapphire')}
-        />
-      }
-      label="Sapphire"
-      sx={{ marginBottom: '-7px' }}
-    />
-    </Box>
-    <Box sx={{marginBottom: '-7px'}}>
-    <FormControlLabel
-      control={
-        <Checkbox
-          checked={selectedParatimes.includes('emerald')}
-          onChange={() => handleParatimesChange('emerald')}
-        />
-      }
-      label="Emerald"
-      sx={{ marginBottom: '-7px' }}
-    />
-    </Box>
-    <Box sx={{marginBottom: '-7px'}}>
-    <FormControlLabel
-      control={
-        <Checkbox
-          checked={selectedParatimes.includes('cipher')}
-          onChange={() => handleParatimesChange('cipher')}
-        />
-      }
-      label="Cipher"
-      sx={{ marginBottom: '-7px' }}
-    />
-    </Box>
-  </Box>
+      <Grid item>
+        <Box>
+          <Typography variant="h6" gutterBottom sx={{ paddingLeft: '-12px' }}>
+            Paratimes
```

Comment:
"ParaTime" is the official capitalization we also use in the docs etc.

```suggestion
            ParaTimes
```